daml-lf/tests/scenario/daml-1.3/contract-keys/EXPECTED.ledger

```diff
@@ -1,8 +1,8 @@
 transactions:
```

Comment:
these are not DAML 1.3 anymore, I'd just move them up a directory and use `dev`.

Reply:
Done.